scripts/sanitycheck

```diff
@@ -570,7 +570,9 @@ class BinaryHandler(Handler):
                        "--log-file="+self.outdir+"/valgrind.log"
                        ] + command
 
+        verbose("Spawning process %s" % " ".join(command))
```

Comment:
Check out quote_sh_list from west for a more faithful way to turn a list of arguments into a string command.

https://github.com/zephyrproject-rtos/west/blob/master/src/west/util.py#L66

Reply:
Thanks, I didn't know `shlex.quote()`. A nice to have here but even better it could solve a real (and unrelated) bug/feature I've been avoiding somewhere else in _get_sub_make().

¿ Anything wrong with the shorter:
```python
" ".join(shlex.quote(el) for el in l)
```

Reply:
No problems with that -- the verbose way it's written is there to aid inserting printlines at each step IIRC, and should probably be cleaned up.